web-wallet/src/routes/(welcome)/setup/restore/NetworkSyncProgress.svelte

```diff
@@ -28,12 +28,12 @@
   {#if !syncStarted || (syncStatus.isInProgress && !syncStatus.progress)}
     <span>Syncing...</span>
   {:else if syncStatus.isInProgress}
-    <span>
-      Syncing: <b
-        >{syncStatus.current.toLocaleString()}/{syncStatus.last.toLocaleString()}</b
-      >
-    </span>
-    <ProgressBar currentPercentage={syncStatus.progress * 100} />
+    <span>Syncing... <b>{syncStatus.progress * 100}%</b></span>
+    <div class="sync-bar">
```

Comment:
Why not making it a real app component then?

Reply:
Done.